api_handlers.go

```diff
@@ -52,7 +51,7 @@ func apiTestRunsHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var browserNames []string
-	if browserNames, err = GetBrowserNames(); err != nil {
+	if browserNames, err = ParseBrowsersParam(r); err != nil {
```

Comment:
@jeffcarp is this special case of multiple-statements-on-one-line-with-error-checking a standard golang idiom?

Reply:
Yep, they're called 'short statements' - https://tour.golang.org/flowcontrol/6